04/app.js

```diff
@@ -2,8 +2,22 @@ const user = {
     firstName: 'Adam',
     lastName: 'Nowak',
     born: {
-        day: '14',
-        month: '04',
+        day: '27',
+        month: '11',
         year: '1985'
     }
-}
\ No newline at end of file
+}
+
+user.Birthday = function() {
+    const now = new Date();
+    const day = this['born']['day'];
+    const month = this['born']['month'];
+    const year = this['born']['year'];
+    const date = new Date(year, month ,day);
+    
+    console.log(now.getMonth())
+    if((now.getDate() == day) && (now.getMonth()+1 == month)) console.log("wszystikego najlepszego!");
```

Comment:
Staraj się używać ścisłego porównania tj. `===` wtedy nie ma problemu z "niejawną konwersją".
Jeśli porównanie nie działa w takim przypadku to musimy sami "przekonwertować" wartości do "wspólnego manownika" np. `Number("01") === 1`